src/nvim/message.c

```diff
@@ -2383,6 +2383,15 @@ static void redir_write(char_u *str, int maxlen)
   char_u      *s = str;
   static int cur_col = 0;
 
+  // Append output to capture().
+  if (capture_ga) {
+    int len = maxlen > 0 ? maxlen : (int)STRLEN(str);
+    ga_grow(capture_ga, len);
+    memmove((char *)capture_ga->ga_data
```

Comment:
`memmove()` does not need a cast at all, especially cast to `char *` as it accepts `void *`. Also this is a job for `memcpy`.